### PR TITLE
[WIP] Fixed golint issues in pool.go

### PR DESCRIPTION
**Pull request by debck**
Signed-off-by: Debasish Sahoo <debasish.sahoo1998@gmail.com>

**Which issue this PR fixes** 
Fixes https://github.com/openebs/openebs/issues/1953

### CODE REVIEW

**Comment by Akash4927** [comment]
Hi @debck, you have to sign your commit. Please have a look at https://github.com/openebs/maya/blob/master/CONTRIBUTING.md

**Comment by Akash4927** [comment]
@debck I think this is because of a different email id. Please use the email registered with your github account.

**Comment by debck** [comment]
I used the registered email as in this PR https://github.com/openebs/node-disk-manager/pull/121
It passed the test

**Comment by Akash4927** [comment]
@debck Please join **hacktoberfest2018** channel on our [slack workspace](https://openebs-community.slack.com/).

**Comment by sonasingh46** [comment]
@debck 
Are you working on this.

**Comment by debck** [comment]
Cant fix the DCO

File: cmd/cstor-pool-mgmt/pool/pool.go [hunk 1]
```diff
@@ -35,12 +35,15 @@ const (
 //PoolAddEventHandled is a flag representing if the pool has been initially imported or created
 var PoolAddEventHandled = false
 
+//It declares a variable string of type string to store the prefix of pool name
```
Comment (kmova) [inline cmd/cstor-pool-mgmt/pool/pool.go]: @debck - check if the lint error is fixed with this comment. Doesn't it require you to start the sentence with variable name?

**Pull request closed without merging**